### PR TITLE
Refactor CSS for PR branch about adding generalized dropdown list options

### PR DESCRIPTION
**Pull request by consideRatio**
I looked through the CSS to understand it better and detected a few typos etc. Here is a screenshot comparing before and after this PR.

Actual differences:
- the radio button was glitchy before
- the text of the dropdown choices was bold before, but now aren't, something I thought aligned with making the label for the options have a normal font weight as well

Before|After
-|-
image|image

### CODE REVIEW

File: kubespawner/spawner.py [hunk 1]
```diff
@@ -1437,67 +1437,68 @@ def _validate_image_pull_secrets(self, proposal):
...
+            }
 
-        #kubespawner-profiles-list .profile .check {
-            display: flex;
-            align-content: center;
-            padding: 12px;
-        }
+            #kubespawner-profiles-list .profile .radio {
+                padding: 12px;
+            }
```
Comment (consideRatio) [inline kubespawner/spawner.py:1454]: I renamed "check" to "radio" as it got me confused it was named "check" as in a checkbox compared to a radiobox we actually use.

File: kubespawner/spawner.py [hunk 2]
```diff
@@ -1437,67 +1437,68 @@ def _validate_image_pull_secrets(self, proposal):
...
-                <div class='options'>
-                    {% for k, option in profile.options.items() %}
-                    <div class='option'>
-                        <label for='option-{{profile.slug}}-{{k}}'>{{option.display_name}}</label>
-                        <select name="option-{{profile.slug}}-{{k}}" class="form-control">
-                            {% for k, choice in option['choices'].items() %}
-                            <option value="{{ k }}" {% if choice.default %}selected{%endif %}>{{ choice.display_name }}</option>
-                            {% endfor %}
-                        </select>
+            {%- for profile in profile_list %}
```
Comment (consideRatio) [inline kubespawner/spawner.py:1472]: I added whitespace chomping to align with how I'm used to doing it in the helm charts, it doesn't matter in practice though, but the emitted HTML is now free of a lot of blank lines.

File: kubespawner/spawner.py [hunk 3]
```diff
@@ -1437,67 +1437,68 @@ def _validate_image_pull_secrets(self, proposal):
...
-            <div class='check'>
-                <input type='radio' name='profile' id='profile-item-{{ profile.slug }}' value='{{ profile.slug }}' {% if profile.default %}checked{% endif %} />
-            </div>
-            <div>
-                <h3>{{ profile.display_name }}</h3>
-                {% if profile.description %}
-                <p>{{ profile.description }}</p>
-                {% endif %}
-                {% if profile.options %}
-                <div class='options'>
```
Comment (consideRatio) [inline kubespawner/spawner.py:1484]: I removed a reference to a class that wasn't defined, `options` with an suffix s.

**Comment by yuvipanda** [comment]
Yay, this looks great @consideRatio!

**Pull request merged**